README.md

```diff
@@ -6,32 +6,32 @@
 
 ## Project setup
 ```
-npm install
+pnpm install
 ```
 
 ### Compiles and hot-reloads for development
 ```
-npm run serve
+pnpm run serve
```

Comment:
You need to let people know about this change.

And also let them know they need to do `npm i -g pnpm` before they can build this project.